dist/raw/ProjectName/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ProjectName",
+  "name": "projectname",
```

Comment:
Shouldn't this be camel case?

Reply:
Nice question!! This was recently updated upstream is the short version. We're installing `@react-native-community/cli` as a devDependency, meaning that `npx` prioritizes that installation over fetching, and since we bumped from 4.x to 5.x we get different results.

https://github.com/react-native-community/cli/pull/1370
https://github.com/react-native-community/cli/blame/84e9f0ba54a391db43ee30f1da2824845f6c289e/packages/cli/src/commands/init/editTemplate.ts#L41